dialog_test/flowtest.py

```diff
@@ -3,13 +3,16 @@
 import flowtest_v1
 import pkg_resources
 import pandas as pd
+import validators
 
 DATA_FOLDER='tests'
 OUTPUT_FOLDER='results'
 USERNAME=os.environ["ASSISTANT_USERNAME"]
 PASSWORD=os.environ["ASSISTANT_PASSWORD"]
+WA_URL=os.environ["ASSISTANT_URL"]
```

Comment:
Use WA_URL = os.environ.get("ASSISTANT_URL",url_default)) and you can remove the rest of the changes.  I would prefer not to add another external package (`validators`)